what4/CHANGES.md

```diff
@@ -42,6 +42,49 @@
   returns the lower and upper version bounds, as determined by querying an
   online SMT solver.
 
+* Add `arrayCopy`, `arraySet`, and `arrayRangeEq` methods to `IsExprBuilder`.
+
+* Add a `getUnannotatedTerm` method to `IsExprBuilder` for retrieving the
+  original, unannotated term out of an annotated term.
+
+* `IsExprBuilder` now has `floatSpecialFunction{,0,1,2}`
+  and `realSpecialFunction{,0,1,2}` methods which allow the use of special
+  values or functions such as `pi`, trigonometric functions, exponentials, or
+  logarithms. Similarly, `IsInterpretedFloatExprBuilder` now has
+  `iFloatSpecialFunction{,0,1,2}` methods. Although little solver support
+  exists for special functions, `what4` may be able to prove properties about
+  them in limited cases.
+  * The `realPi`, `realLog`, `realExp`, `realSin`, `realCos`, `realTan`,
+    `realSinh`, `realCosh`, `realTanh`, and `realAtan2` methods of
+    `IsExprBuilder` now have default implementations in terms of
+    `realSpecialFunction{,0,1,2}`.
+
+* Add an `exprUninterponstants` method to `IsSymExprBuilder` which returns the
```

Comment:
typo?

Reply:
Oops, the C key on my keyboard must have given out at the wrong moment. Fixed.